includes/class-cache.php

```
@@ -376,10 +405,241 @@ public function combine_css() {
+			if ( ! $fs || ! $fs->exists( $handle ) ) {
+				return false;
+			}
+
+			$contents = $fs->get_contents( $handle );
+			if ( false === $contents ) {
+				return false;
+			}
+
+			$stored = json_decode( $contents, true );
+			return is_array( $stored ) && $stored === $eligible_handles;
+		}
```

PHPMD (2.15.0)

[error] The method combined_handles_match is not named in camelCase. (undefined)

(CamelCaseMethodName)